es/views.rst

```diff
@@ -1,14 +1,817 @@
+        }
+    }
+
+Y después en el elemento podemos acceder al modelo de publicaciones
+paginado. Para obtener las últimas 5 publicaciones en una lista
+ordenada tendríamos que hacer algo similar a lo siguiente:
+
+.. code-block:: php
+
+    <h2>Latest Posts</h2>
+    <?php
+      $posts = $this->requestAction(
+        'posts/index/sort:created/direction:asc/limit:5'
+      );
+    ?>
+    <ol>
+    <?php foreach ($posts as $post): ?>
+          <li><?php echo $post['Post']['title']; ?></li>
+    <?php endforeach; ?>
+    </ol>
+
+Almacenar elementos
+-------------------
+
+Puedes tomar ventaja de el almacenamiento del almacenamiento de
+vistas de CakePHP si provees un parametro de cache. Si se le
+asigna true almacenará el elemento en el cache 'default'
+(predeterminado) de la configuración. De lo contrario puedes
+configurar cuál cache deberá usarse en la configuración. Ver
+:doc:`/core-libraries/caching` para más información de cómo
+configurar la clase :php:class:`Cache`. Un ejemplo simple de
+almacenamiento un elemento sería así::
+
+    echo $this->element('helpbox', array(), array('cache' => true));
+
+Si renderizas un mismo elemento más de una vez en una vista y tienes
+activado el almacenamiento, asegurate de asignarle a 'key' un valor
+diferente cada vez. Esto prevendrá que cada llamada sobreescriba
+el resultado anterior de las llamadas anteriores de element(). Por
+ejemplo::
+
+    echo $this->element(
+        'helpbox',
+        array('var' => $var),
+        array('cache' => array('key' => 'primer_uso', 'config' => 'view_long')
+    );
+
+    echo $this->element(
+        'helpbox',
+        array('var' => $varDiferente),
+        array('cache' => array('key' => 'segundo_uso', 'config' => 'view_long')
+    );
+
+Lo de arriba asegurará que los elementos se almacenen por separado. Si
+quieres que todo almacenamiento use la misma configuración, puedes evitar
+un poco de repetición usando :php:attr:`View::$elementCache` a la
+configuración de cache que quieres usar. CakePHP usará esta configuración
+cuando no se provea una.
+
+Solicitar elementos de un plugin
+--------------------------------
+
+2.0
+---
+
+Para cargar un elemento de un plugin, usa la opción de `plugin` (movida fuera
+la opción de `data` en la versión 1.x)::
+
+    echo $this->element('helpbox', array(), array('plugin' => 'Contacts'));
+
+2.1
+---
+
+Si estás usando algún plugin y deseas usar elementos dentro del plugin
+simplemente utiliza :term:`plugin syntax`. Si la vista está siendo
+renderizada por un controlador o acción de plugin, el nombre del plugin
+será antepuesto automáticamente a todos los elementos usados, a menos que
+otro nombre de un plugin esté presente. Si el elemento no existe en el
+plugin, buscará en el directorio principal APP::
+
+    echo $this->element('Contacts.helpbox');
+
+Si tu vista es parte de un plugin, puedes omitir el nombre del plugin.
+Por ejemplo, si estás en ``ContactsController`` del plugin de Conctacts::
```

Comment:
@clinchergt you missed one, but was an edit from an existing comment so not obvious... :wink: 
`Contacts` instead of `Conctacts`